java-file-formats/word-library/working-with-bookmarks.md

```diff
@@ -406,12 +407,12 @@ As per Microsoft Word behavior, you cannot replace the bookmark contents when th
 
 Case 1
 
-![Bookmark start and end present in different tables](WorkingwithBookmarks_images/WorkingwithBookmarks_img1.jpeg)
+![Bookmark start and end present in different tables](WorkingwithBookmarks_images/java-file-formats-word-library-different-tables.jpeg)
```

Comment:
Change the Image Folder name casing to small case. Update the changes in all needed places.

Reply:
Changed